ethernet: Support querying `min_mtu` and `max_mtu`

cathay4t opened the pull request:
Including `min_mtu` and `max_mtu` to `BaseInterface` when querying.
Ignore them when applying and verifying.

Raise pre-apply error when desired MTU not in the supported range.

Python binding updated by adding constants of `Interface.MIN_MTU` and
`Interface.MAX_MTU`.

Integration test cases included.

ffmancera commented on tests/integration/veth_test.py:
```diff
@@ -414,3 +414,32 @@ def test_new_veth_with_ipv6_only():
+                ]
+            }
+        )
+
```
2 blank lines are required between tests, thanks!

cathay4t replied:
Fixed. Thanks!

cathay4t committed:
ethernet: Support querying `min_mtu` and `max_mtu`

Including `min_mtu` and `max_mtu` to `BaseInterface` when querying.
Ignore them when applying and verifying.

Raise pre-apply error when desired MTU not in the supported range.

Python binding updated by adding constants of `Interface.MIN_MTU` and
`Interface.MAX_MTU`.

Integration test cases included.

Signed-off-by: Gris Ge <fge@redhat.com>
